### PR TITLE
Semi finish

### PR DESCRIPTION
**Pull request by DilsherB**
## Screen Shots
data inserted into specializations
data inserted into vets
data inserted into visits
specializations table created
table vets created
visits table created
query latest visit to william tatcher
image
image
image
image
image
image
image
image

### Schema
1. three new tables created.
   a: one "vets"
   b: two `join tables` "specializations" and "visits".

### Data
inserts data in "vets", "specializations" and "visits".

### Queries
queries for the followings are written

1.     Who was the last animal seen by William Tatcher?
2.     How many different animals did Stephanie Mendez see?
3.     List all vets and their specialties, including vets with no specialties.
4.     List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
5.     What animal has the most visits to vets?
6.     Who was Maisy Smith's first visit?
7.     Details for most recent visit: animal information, vet information, and date of visit.
8.     How many visits were with a vet that did not specialize in that animal's species?
9.     What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -109,12 +109,12 @@ JOIN animals A ON A.id = Vi.animal_id
...
-JOIN vets V ON V.id = Vi.vet_id
-JOIN animals A ON A.id = Vi.animal_id
-WHERE V.name = 'Stephanie Mendez'
-GROUP BY V.name;
+-- SELECT V.name AS vet, COUNT(A.name)
+-- FROM visits Vi
+-- JOIN vets V ON V.id = Vi.vet_id
+-- JOIN animals A ON A.id = Vi.animal_id
+-- WHERE V.name = 'Stephanie Mendez'
+-- GROUP BY V.name;
```
Comment (franclobo) [inline queries.sql]: - [OPTIONAL] Unless the commented code help to the reader we should remove those lines of code that we are not gonna use, remember YAGNI.

File: queries.sql [hunk 2]
```diff
@@ -160,24 +160,24 @@ FULL JOIN specializations SP ON SP.vet_id = v.id
...
+-- SELECT vet_name,
+--        COUNT(vet_name) AS num_visits
+-- FROM (
+--     SELECT v.name AS vet_name
+--     FROM visits vi
+--     FULL JOIN vets v ON v.id = vi.vet_id
+--     FULL JOIN specializations sp ON sp.vet_id = v.id
+--     WHERE sp.vet_id IS NULL
+-- ) AS t
+-- GROUP BY vet_name;
```
Comment (franclobo) [inline queries.sql]: - [OPTIONAL] Unless the commented code help to the reader we should remove those lines of code that we are not gonna use, remember YAGNI.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @DilsherB,

# STATUS: APPROVED 😃 

git-merge


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correct data ✔️ 
- Correct queries ✔️ 
- Correct schema ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (DilsherB) [reply queries.sql]: done

File: queries.sql (hunk 2 above)
Comment (DilsherB) [reply queries.sql]: done

**Pull request merged**